websites/D/Diziroll/presence.ts

```diff
@@ -0,0 +1,61 @@
+const presence = new Presence({ clientId: "818550584994168934" });
+
+let stream: { duration: number; currentTime: number; paused: boolean };
+presence.on("iFrameData", (data: { duration: number; currentTime: number; paused: boolean }) => (stream = data));
+
+presence.on("UpdateData", async () => {
+  const path: string = document.location.pathname,
+  presenceData: PresenceData = {largeImageKey: "diziroll"};
+
+  if(path.startsWith("/arsiv")){
+  presenceData.details = "Bir sayfaya bakıyor:";
+  presenceData.state = "Arşiv";
+  presenceData.startTimestamp = Date.now();
+
+  }else if(path.startsWith("/listeler")){
+  presenceData.details = "Bir sayfaya bakıyor:";
+  presenceData.state = "Listeler";
+  presenceData.startTimestamp = Date.now(); 
+  
+  }else if(path.startsWith("/hesabim")){
+      presenceData.details = "Bir sayfaya bakıyor:";
+      presenceData.state = "Hesabım";
+      presenceData.startTimestamp = Date.now();
+
+  }else if(path == "/"){
+      presenceData.details = "Bir sayfaya bakıyor:";
+      presenceData.state = "Ana Sayfa";
+      presenceData.startTimestamp = Date.now();
+
+  }else if(document.getElementById("archive-page")){
+      presenceData.details = "Bir dizi türünü inceliyor: ";
+      presenceData.state = document.querySelector("div.title").innerText;
+      presenceData.startTimestamp = Date.now();
+
+  }else if(document.getElementById("series-page")){
+      presenceData.details = "Bir diziyi inceliyor:";
+      presenceData.state = document.querySelector("div.top > h1").innerText;
+      presenceData.startTimestamp = Date.now();
```

Comment:
just set startTimestamp once.. and I think you need to MathFloor it

Reply:
https://github.com/PreMiD/Presences/pull/3220/commits/cf692a8d65eb2dfe1877c4bcc7db3ebd77e6d42b